Resources/doc/usage.rst

```diff
@@ -0,0 +1,93 @@
+Usage
+=====
+
+Manager
+-------
+
+Elasticsearch bundle to communicate with elasticsearch index, it provides managers that could handle several indexes.
```

Comment:
Elasticsearch bundle provides managers that could handle several indexes to communicate with elasticsearch.